src/js/components/App.js

```diff
@@ -12,18 +12,15 @@ import ErrorNotice from "./ErrorNotice"
 import HTMLContextMenu from "./HTMLContextMenu"
 import Preferences from "./Preferences/Preferences"
 import Prefs from "../state/Prefs"
+import SpaceModal from "./SpaceModal"
 import View from "../state/View"
 import brim from "../brim"
-import refreshSpaceNames from "../flows/refreshSpaceNames"
-import SpaceModal from "./SpaceModal"
 
 export default function App() {
   brim.time.setZone(useSelector(View.getTimeZone))
   brim.time.setDefaultFormat(useSelector(Prefs.getTimeFormat))
-  let dispatch = useDispatch()
 
   useEffect(() => {
-    dispatch(refreshSpaceNames())
```

Comment:
This happens in the initializer now before react boots up: https://github.com/brimsec/brim/pull/875/files#diff-0b427a5466e0c7a032be98e7a3e28501R50